lib/bolognese/utils.rb

```diff
@@ -1369,5 +1373,20 @@ def hsh_to_fos(hsh)
           "lang" => hsh["lang"] }.compact]
       end
     end
+
+    def dfg_ids_to_fos(dfg_ids)
+      dfgs = JSON.load(File.read(File.expand_path('../../../resources/oecd/dfg-mappings.json', __FILE__))).fetch("dfgFields")
```

Comment:
I wonder if it's worth defining a constant for the resources directory and using that, to make the directory path a little cleaner (and potentially more portable if we ever need to move it?).

Similar to the spec fixtures:
https://github.com/datacite/bolognese/blob/master/spec/spec_helper.rb#L36-L38
https://github.com/datacite/bolognese/blob/master/spec/datacite_utils_spec.rb#L180

Reply:
Can do